rotkehlchen/chain/evm/types.py

```diff
@@ -132,3 +133,9 @@ def asset_id_is_evm_token(asset_id: str) -> tuple[ChainID, ChecksumEvmAddress] |
 
     address = string_to_evm_address(match.group(3))
     return chain_id, address
+
+
+class RemoteDataQueryStatus(Enum):
+    FAILED = auto()
+    NEW_DATA_FETCHED = auto()
```

Comment:
Just to keep the names shorter

```suggestion
    NEW_DATA = auto()
```